tests/tel2puml/find_unique_graphs/test_ingest_data.py

```diff
@@ -0,0 +1,81 @@
+"""Tests for ingest_data module."""
+
+import yaml
+from pathlib import Path
+
+from tel2puml.find_unique_graphs.otel_ingestion.ingest_otel_data import (
+    IngestData,
+)
+from tel2puml.find_unique_graphs.otel_ingestion.otel_data_model import (
+    IngestDataConfig,
+    NodeModel,
+)
+from tel2puml.find_unique_graphs.otel_ingestion.otel_data_holder import (
+    SQLDataHolder,
+)
+from tel2puml.find_unique_graphs.otel_ingestion.otel_data_source import (
+    JSONDataSource,
+)
+
+
+class TestIngestData:
+    """Collection of tests for IngestData class."""
+
+    @staticmethod
+    def test_integration_sql_json(
```

Comment:
should call this test `test_load_data_to_data_holder`

Reply:
Updated